packages/flutter/test/cupertino/date_picker_test.dart

```diff
@@ -2414,6 +2418,72 @@ void main() {
 
     expect(find.byType(CupertinoPickerDefaultSelectionOverlay), findsExactly(4));
   });
+
+  testWidgets('CupertinoDatePicker month column width validation in Portuguese', (WidgetTester tester) async {
```

Comment:
This test can be a bit brittle. I would expose `_getColumnWidth` as visibleForTesting and test the implementation instead.

Reply:
or expose a test-only method to set the text in tests instead of depending on localization and the material font